src/Microsoft.DocAsCode.Build.Engine/HostServiceCreatorWithIncremental.cs

```diff
@@ -111,5 +116,29 @@ private void PostCreate(HostService hostService, IEnumerable<FileAndType> files)
                 IncrementalContext.ReportModelLoadInfo(hostService, loadedFiles, BuildPhase.PreBuildBuild);
             }
         }
+
+        private bool ShouldProcessorTraceInfoCore(IDocumentProcessor processor)
+        {
+            if (!(processor is ISupportIncrementalDocumentProcessor))
+            {
+                Logger.LogVerbose($"Processor {processor.Name} cannot suppport incremental build because the processor doesn't implement {nameof(ISupportIncrementalDocumentProcessor)} interface.");
```

Comment:
update the suppport to support? :)

---
In reply to: [91652735](https://github.com/dotnet/docfx/pull/982#discussion_r91652735) [](ancestors = 91652735)